playbooks/aws/full_deploy_and_configure.yml

```diff
@@ -27,6 +27,20 @@
     - cisco.catalystwan.onboarding_controllers
 
 
+- name: Cluster configuration
```

Comment:
I think it would be better to create second playbook/directory that would store cluster configuration, so users would take either regular config, or cluster config, and apply them.

@sashiv20 what do you think, what would be best way to store and reuse these configurations?

Reply:
yes. for cluster, lets create a new playbook - i believe one for azure and one for aws.

Reply:
Got it. I've created new playbooks for cluster deployment.